### PR TITLE
Fix/persistent store

### PR DESCRIPTION
**Pull request by aphelionz**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,7 +27,8 @@
     "level-js": "~4.0.1",
```
Comment (shamb0t) [inline package.json]: Can `leveljs` and `leveldown` be deleted?

File: index-browser.js [hunk 2]
```diff
@@ -1,3 +1,2 @@
-const level = require('level-js')
```
Comment (shamb0t) [inline index-browser.js]: Seems these files can be deleted and replaced with a single index file

Comment (aphelionz) [reply index-browser.js]: Done and done!

File: package.json (hunk 1 above)
Comment (aphelionz) [reply package.json]: Yes, they can

**Comment by shamb0t** [comment]
:tada:

**Pull request merged**